### PR TITLE
TC03 - User cannot log into Railway with invalid password

### PR DESCRIPTION
**Pull request by thienan090196**

### CODE REVIEW

File: src/test/java/com/logigear/tests/LoginTest.java [hunk 1]
```diff
@@ -35,4 +36,15 @@ public void TC02() {
         String expectedResult = ErrorMessage.LOGIN_ERROR;
         Assert.assertEquals(actualResult, expectedResult, "Error message fails to display!");
     }
+
+    @Test(description = "User cannot log into Railway with invalid password ")
+    public void TC03() {
+        account = new Account(Constant.EMAIL, DataHelper.getRandomPassword());
```
Comment (tanmle) [inline src/test/java/com/logigear/tests/LoginTest.java:42]: why do we need random password for an invalid case?

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/test/java/com/logigear/tests/LoginTest.java (hunk 1 above)
Comment (thienan090196) [reply src/test/java/com/logigear/tests/LoginTest.java:42]: If we don't use random, we only use one type of password maybe like blank string or any defined password

File: src/test/java/com/logigear/tests/LoginTest.java [hunk 2]
```diff
@@ -35,4 +36,15 @@ public void TC02() {
...
+        homePage.goToLoginPage();
+        loginPage.login(account);
+
+        String actualResult = loginPage.getErrorMessage();
+        String expectedResult = ErrorMessage.LOGIN_ERROR;
```
Comment (fey031) [inline src/test/java/com/logigear/tests/LoginTest.java:47]: if you define this message in Constant, don't need declare new variable for it here

Comment (thienan090196) [reply src/test/java/com/logigear/tests/LoginTest.java]: I updated my code due to your review, please review it for me

File: src/main/java/com/logigear/helpers/DataHelper.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.logigear.helpers;
+
+import com.github.javafaker.Faker;
+
+public class DataHelper {
+
+    private static Faker faker = new Faker();
+
+    public static String getRandomPassword() {
```
Comment (tanmle) [inline src/main/java/com/logigear/helpers/DataHelper.java:9]: if you don't use this method anymore then remove it.

**Review by tanmle — CHANGES_REQUESTED** [review]

**Review by tanmle — APPROVED** [review]

**Review by fey031 — APPROVED** [review]
LGTM

**Pull request merged**